bowling-ball/Game.cs

```diff
@@ -4,16 +4,56 @@ namespace BowlingBall
 {
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
+        int[] Rolls = new int[21];
+
+        public void Roll(int[] pins)
+        {
+            int i=0;
+            foreach(int iterator in pins)
+            {
+                Rolls[i]=iterator;
+                i=i+1;
+            }
         }
 
-        public int GetScore()
+        public int GetScore()

```

Comment:
Always add comments for functions what it is doing and what is expected from this function.